faq.md

```diff
@@ -318,23 +318,42 @@ virtual machine are implemented in C/C++.
 
 ## Why did Flutter choose to use Dart?
 
-Dart gives us a scalable language and a feature set that supports simple scripts
-all the way up to full-featured apps. We wanted something that would be
-instantly familiar to JavaScript, Java, and C# developers. Dart's robust core
-libraries meant that we didn't need to write a lot of core library code.
-
-We also wanted our apps to be extremely fast, and in our experience the Dart VM
-has very impressive performance. We also really liked the Dart VM's other
-features like the Observatory (a real-time VM profiler, debugger, and
-introspection tool) and snapshots (a binary form of the app that's quick to
-load).
-
-Very important was the requirement to run fast on iOS. The Dart VM supports an
-ahead-of-time compilation mode, meaning we can compile Dart code to native code
-and ship a Flutter app to Apple's App Store.
-
-In short, we adopted Dart because it was the best language that we could find
-that we wanted to code in every day.
+The primary criteria we used to pick a programming language were the following:
+
+* _Predictable, high performance_.  With Flutter, we want to empower developers
+  to create fast, fluid user experiences.  In order to achieve that, we need to
+  be able to run a significant amount of end-developer code during every
+  animation frame.  That means we need a language that both delivers high
+  performance and that delivers that performance predictably, without periodic
```

Comment:
I would probably suggest "both delivers high performance and does so predictably".